### ISSUE TITLE
Documentation updates

### ISSUE THREAD

**Issue by MSevey** [SkynetLabs/skynet-js#540]
Background

I'm thinking more and more that we should have separate docs for each language. There are too many differences in functionality between them all at this point. 

It made sense to have a single Slate page in the beginning when: 

The SDKs were in sync.

The SDKs were mostly just thin wrappers around curl calls.

But, at this point:

The complexity of the SDKs has grown and the format no longer works.

skynet-js is much different than the others due to being a browser SDK (and having more functionality).

Users have been contributing things they needed to the SDKs they were using, which we didn't have the resources to port to the other SDKs.

SkynetLabs has been adding things to the node SDK as it was the main server-side SDK we've been using, so it's even more out of sync than the other ones.

All of this is making for a confusing experience when reading the docs. I've been updating them recently and it's rough having to add caveats all over the place like "this only works in SDK X".

Proposal

Therefore I propose that we have separate pages for all the docs.

I would make sdk.skynetlabs.com point to skynet-js docs. The current docs were written with the browser in mind, so we can use them mostly as-is, just scrubbing references to other langs.

sdk.skynetlabs.com can link out to the other docs. Not sure where they should live yet. Maybe at sdk.skynetlabs.com/<lang>?

I would try to make the other docs auto-generated. You can auto-generate really good docs for Rust, and I think similar tooling exists for the other langs. This would help keep the docs up-to-date, which is another thing we've been struggling with.

I would also remove the doc versioning. It was a good idea, but people haven't used it, it made the docs more complicated, and we are now committed to backwards compatibility.

TL;DR

There's been significant drift in the functionality of the SDKs for several reasons.

The docs are confusing and littered with caveats about "this only applies to language X". They should be separate for the best experience.

We can auto-generate docs for each language using language-specific tooling.

Having good, up-to-date SDK docs would reflect well on the company and its products.
